test/unit/migrations/migrator-test.js

```diff
@@ -35,6 +37,23 @@ const firstTimeState = {
   meta: { version: 0 },
   data: require('../../../app/scripts/first-time-state'),
 }
+describe('liveMigrations require list', () => {
+  it('should include all the migrations', async () => {
+    const fileNames = await pify(cb => fs.readdir('./app/scripts/migrations/', cb))()
+    const migrationNumbers = fileNames.reduce((agg, filename) => {
+      const name = filename.split('.')[0]
+      if (/^\d+$/.test(name)) {
+        agg.push(name)
+      }
+      return agg
+    }, []).map((num) => parseInt(num))
+
+    migrationNumbers.forEach((num) => {
+      const migration = liveMigrations.find((m) => m.version === num)
+      assert(migration, `migration should be include in the index missing migration ${num}`)
```

Comment:
`migration should be include in the index missing migration ${num}` would be easier to read with a colon-

`migration should be include in the index: missing migration ${num}`

Reply:
Grammatically a semicolon seems more appropriate, but yeah I agree that a separator of some sort seems warranted.